### PR TITLE
Side Navigation now pulling in Archive Event Page correctly

### PR DESCRIPTION
**Pull request by kave**
## Changes

- Checking for class instances of Browse & BrowseFilterable Pages for Side Navigation
- Fixed events pages so it only renders non-archived events

## Testing

- load latest refresh DB
- visit `http://content.localhost:8000/about-us/events2/`

## Review

- @kurtw 
- @richaagarwal 

## Screenshots

screen shot 2016-03-15 at 8 48 42 am

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/v1/models/browse_filterable_page.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def get_context(self, request, *args, **kwargs):
 
 
     def get_form_class(self):
-        return filterable_context.get_form_class(self)
+        return filterable_context.get_form_class()
```
Comment (richaagarwal) [inline cfgov/v1/models/browse_filterable_page.py]: woops, that was my bad. thanks for fixing!

File: cfgov/v1/util/filterable_context.py [hunk 2]
```diff
@@ -61,8 +63,10 @@ def get_form_specific_filter_data(page, form_class, request_dict):
                     request_dict.get(request_field_name, '')
     return filters_data
 
+
 # Returns a queryset of AbstractFilterPages
 def get_page_set(page, form, hostname):
+    from ..models.browse_filterable_page import EventArchivePage, BrowseFilterablePage
     # If this is the Newsroom, then we need to go get the blog
```
Comment (richaagarwal) [inline cfgov/v1/util/filterable_context.py]: is `BrowseFilterablePage` referenced in this function?

File: cfgov/v1/models/browse_filterable_page.py (hunk 1 above)
Comment (kave) [inline cfgov/v1/models/browse_filterable_page.py]: lol no problem

File: cfgov/v1/util/filterable_context.py (hunk 2 above)
Comment (kave) [inline cfgov/v1/util/filterable_context.py]: ah nah, i'll remove it. thanks!

**Comment by kurtw** [comment]
:+1:

**Comment by richaagarwal** [comment]
:+1:

**Pull request merged**